### ISSUE TITLE
Problem with animating color of placeholder in TextInput by interpolateColors.

### ISSUE THREAD

**Issue by wiru123321** [software-mansion/react-native-reanimated#2273]
## Description
If i want animate the color of placeholder TextInput i get weird color on output. It works bad only when i want animate this, when i put exact colors for props placeholderTextColor its render great.

### Expected behavior
Color should change the value. Like in the example without animation.

### Actual behavior & steps to reproduce
The behavior are presented in the snack.

## Snack or minimal code example
https://snack.expo.dev/G7GThm8eu

## Package versions
"react-native-reanimated": "~2.2.0",
"react": "~17.0.0",
"react-native": "0.64.1",

## Affected platforms

- [ X ] iOS

**Comment by wiru123321** [comment]
Is any idea how to fix this?

**Comment by wszydlak** [comment]
@jakub-gonet any updates on this? We have this issue using Reanimated-2, don't know why someone removed Reanimated2 label.

**Comment by jakub-gonet** [comment]
No, I haven't had time to look at it. In the code you're using Reanimated 1 (even with Reanimated v2.2.0), that's why the label was replaced.

**Comment by wszydlak** [comment]
Found a solution:

```tsx
const AnimatedTextInput = Animated.createAnimatedComponent(TextInput, {
  setNativeProps(ref, props) {
    if (props.placeholderTextColor) {
      const { placeholderTextColor } = props;
      const a = (placeholderTextColor >>> 24) / 255;
      const r = (placeholderTextColor << 8) >>> 24;
      const g = (placeholderTextColor << 16) >>> 24;
      const b = (placeholderTextColor << 24) >>> 24;

      const color = convertToHext({r,g, b, a});

      ref.setNativeProps({ placeholderTextColor: color });
    }
  },
});
```

But seems this should be included in reanimated.

**Comment by wszydlak** [comment]
Found yet another solution, if you add `placeholderTextColor` to whitelistedNativeProps it starts working without any hacks:

```js
Animated.addWhitelistedNativeProps({
  placeholderTextColor: true,
});
```

I prepared PR that will fix this issue: #2304
